community/security/src/main/java/org/neo4j/server/security/auth/UserSerialization.java

```diff
@@ -72,21 +72,28 @@
     private String serialize( User user )
     {
         return join( userSeparator, user.name(),
+                user.group(),
```

Comment:
If group was previously not part of the auth file, perhaps we should not add it, but add the groups file? That would remove a potentially unclean intermediate state.